File: Problems/TwoSum.php
class Solution {

    /**
     * @param Integer[] $nums
     * @param Integer $target
     * @return Integer[]
     */
    function twoSum($nums, $target) {
        foreach($nums as $i=>$v){
            foreach($nums as $i1=>$v1){
                if(($v+$v1)==$target && $i1!=$i){
                    return [$i,$i1];
                }
            }  
        }
    }
}

File: DataStructure/TwoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        result = []
        for i in range(len(nums)):
            aux = target - nums[i]
            if aux in nums and nums.index(aux) != i:
                result.append(i)
                result.append(nums.index(aux))
                return result
